_posts/2023-06-27-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.26
+author: Priya
+date: 2023-06-27
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+### Improved Date difference rule
+
+Enhanced the Date difference rule to ensure that the result of the date difference is always a positive integer when the start date is earlier than the end date.
+
+
+### Defect fixes
+
+* Resolved the issue where the "Move to" UI in the Folder feature was broken.
+* Fixed the issue where not all assets of an account were transferred to the target user.
+* Resolved the issue of invalid HTML presence in the DOM, causing the element to overflow and be absent from the viewport.
```

Comment:
Resolved the issue of element overflowing the viewport on <Data Library?>

Reply:
fixed